### PR TITLE
[13.0][MIG] base_cron_exclusion

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Standard migration.

@ForgeFlow

### CODE REVIEW

File: base_cron_exclusion/README.rst [hunk 1]
```diff
@@ -47,10 +44,10 @@ To use this module, you need to:
 Bug Tracker
 ===========
 
-Bugs are tracked on `GitHub Issues <https://github.com/OCA/server-tools/issues>`_.
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/sever-tools/issues>`_.
```
Comment (MiquelRForgeFlow) [inline base_cron_exclusion/README.rst]: something is wrong

File: base_cron_exclusion/i18n/base_cron_exclusion.pot [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* base_cron_exclusion
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
```
Comment (MiquelRForgeFlow) [inline base_cron_exclusion/i18n/base_cron_exclusion.pot:7]:
```suggestion
"Project-Id-Version: Odoo Server 13.0\n"
```

File: base_cron_exclusion/README.rst (hunk 1 above)
Comment (LoisRForgeFlow) [reply base_cron_exclusion/README.rst]: Fixed!

File: base_cron_exclusion/i18n/base_cron_exclusion.pot (hunk 2 above)
Comment (LoisRForgeFlow) [reply base_cron_exclusion/i18n/base_cron_exclusion.pot:7]: pot file will be updated after merge.

**Comment by LoisRForgeFlow** [comment]
@MiquelRForgeFlow could you update your view?

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by HviorForgeFlow — APPROVED** [review]

**Review by JordiBForgeFlow — APPROVED** [review]

**Pull request merged**